webservices/config.py

```diff
@@ -19,14 +19,15 @@ def get_cycle_end(year):
     return year if year % 2 == 0 else year + 1
 
 CURRENT_YEAR = datetime.datetime.now().year
+CURRENT_CYCLE = datetime.datetime.now().year + datetime.datetime.now().year % 2
 
 SQL_CONFIG = {
     'START_YEAR': get_cycle_start(1980),
     'START_YEAR_AGGREGATE': get_cycle_start(2008),
     'END_YEAR_ITEMIZED': get_cycle_start(CURRENT_YEAR),
     'CYCLE_END_YEAR_ITEMIZED': get_cycle_end(CURRENT_YEAR),
     'PARTITION_START_YEAR': 1978,
-    'PARTITION_END_YEAR': 2018,
+    'PARTITION_END_YEAR': CURRENT_CYCLE,
```

Comment:
Ah I see what you're doing, :clap:!  That said, is there anything we need to account for on the DB side to make sure the app doesn't suddenly break on account of it referencing a partition child under the hood that doesn't exist yet?

In other words, let's say we're now in the 2020 cycle.  Right now, the DB won't automatically create this, we need to run the [`add_itemized_partition_cycle`](https://github.com/18F/openFEC/blob/develop/manage.py#L295-L321) management command before those tables exist.

I just want to make sure we're not creating a situation where the user would suddenly see an option that would throw an error!

Reply:
We should already have the next cycle tables built by January 1, 2020 or we would start getting insertion errors relatively quickly.  I would be fine making those tables now if you are concerned about it. 

We can also set the defaults on the web app separately from the API. In the future, we might want to hand code that value for a while to create more interesting presentation defaults, but I would rather have things work by default and over ride them later if people want something different. I feel like we had a lot of hand coded things to change this cycle that were not ideal and took us months to track down and change.